src/index.js

```diff
@@ -53,12 +53,15 @@ function installRouterPrefetch(
         default: 2000
       }
     },
+    setup(props, context) {
```

Comment:
`RouterLink` now exposes a `setup()` method, so we need to call it in order to have full inheritance.
Otherwise our custom `RouterLink` component won't render anything.